### ISSUE TITLE
Unnecessary duplicate entries for CCC Entry

### ISSUE THREAD

**Issue by kwarkster** [harrisms2/dataish#2]
Some clients exit Rapid Rehousing, and are then showing multiple re-entries to CCC Point of entry.  The multiple re-entries are really just caseworkers entering new updates as they contact the client trying to figure out where he/she has gone since leaving RR . You don't want a new record for each of f these instances, you just want one line with the earliest CCC Point of entry/CC entry date.  Look at Client id 2879 for an example of what I'm talking about.

**Comment by harrisms2** [comment]
Good catch. So, for example, if exit_rr1 < exit_rr2 < min_entry_cc then we don't consider exit_rr1 to be a case of recidivism. I will work on fixing this.
